lib/pbench/server/api/resources/intake_base.py

```diff
@@ -312,12 +305,7 @@ def _intake(
 
             # Now we're ready to pull the tarball, so ask our helper for the
             # length and data stream.
-            try:
-                access = self._access(intake, request)
-            except APIAbort:
-                raise
-            except Exception as e:
-                raise APIInternalError(str(e)) from e
+            access = self._stream(intake, request)
```

Comment:
I like `_stream` much better than `_access`...but you didn't want to rename the local variable to match?

Reply:
I don't *not* want to ...